lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -1098,10 +1098,11 @@ LogicalResult ConvertAtenOp<AtenPowTensorTensorOp>::matchAndRewrite(
       cast<TensorType>(getTypeConverter()->convertType(op.getType()));
 
   Value expTensor = adaptor.getExponent();
-  if (expTensor.getType() != selfTy) {
+  auto expTensorTy = cast<RankedTensorType>(expTensor.getType());
```

Comment:
Is the expTensor always ranked?

(the check that self is ranked a few lines above seems wrong btw, should probably be a dyn_cast)

Reply:
Good catch, I will adjust